po/zh-CN.po

```diff
@@ -2024,7 +2024,7 @@ msgstr ""
 #: src/welcome-day-1.md:24
 msgid ""
 "The questions will likely mean that we talk about things ahead of the slides."
-msgstr "有些问题会导致我们提前谈到后面的内容"
+msgstr "有些问题会导致我们提前谈到后面的内容."
```

Comment:
use "。"

Reply:
<img width="941" alt="image" src="https://github.com/google/comprehensive-rust/assets/11788119/b43559c7-57be-4116-9f09-a8267919402f">

If you use Poedit locally, you'll be prompted to make this change, or you'll be warned of an error!

Reply:
Ah, interesting! That looks like a problem with Poedit and we should ignore the error in this case.

I created https://github.com/vslavik/poedit/issues/809 for this in the Poedit repository. Perhaps they have a workaround.

Reply:
Hi @SketchK, could you reply on https://github.com/vslavik/poedit/issues/809 to explain what you did to get the warning?

Reply:
> Hi @SketchK, could you reply on [vslavik/poedit#809](https://github.com/vslavik/poedit/issues/809) to explain what you did to get the warning?

OK, Sir, I write something in that issue @mgeisler

Reply:
@SketchK, please apply the suggestion by @kongy above so we can resolve this comment.